python/parser.js

```diff
@@ -22,6 +22,7 @@ function checkSettings(cs) {
     if (cs.floatingRegions) options += floatingRegions(cs.floatingRegions)
     if (cs.accessibilityRegions) options += accessibilityRegions(cs.accessibilityRegions)
     if (cs.layoutRegions) options += layoutRegions(cs.layoutRegions)
+    if (cs.layoutBreakpoints) options += layoutBreakpoints(cs.layoutBreakpoints)
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8717150)
```suggestion
    if (cs.layoutBreakpoints) {options += layoutBreakpoints(cs.layoutBreakpoints)}
```